composites/Plugin/DashboardWidget/components/WordpressFeed.js

```diff
@@ -73,8 +73,10 @@ const WordpressFeedFooter = styled.div`
 
 const WordpressFeedListItem = ( props ) => {
 	return (
-		<WordpressFeedListItemContainer>
+		<WordpressFeedListItemContainer
```

Comment:
is technically WordPress, not sure how impactful it is to change this, but if it's easy to change, please do :)

Reply:
I see this was not introduced in this pull, so I won't block it based on it. Though we might want to change it in the future.